lib/src/utf16.dart

```diff
@@ -13,7 +13,7 @@ import 'package:ffi/ffi.dart';
 ///
 /// [Utf16] is respresented as a struct so that `Pointer<Utf16>` can be used in
 /// native function signatures.
-class Utf16 extends Struct<Utf16> {
```

Comment:
Please update `CHANGELOG.md` describing what this change is doing.

Reply:
Also consider updating `pubspec.yaml` to have a sdk constraint with `2.6.0`

Reply:
Postponing updating 2.6.0 to when we do pub release for this package and the tflite_native package.